addon/components/bs-form/element.js

```diff
@@ -218,7 +213,7 @@ export default class FormElement extends FormGroup {
    * @type string
    * @public
    */
-  @defaultValue
+  @arg
```

Comment:
Can't we change all arguments, that have either no initializer or `null`, to named args, i.e. remove them here from the class (keeping just the DocBlock) and use them as `@...` in the template? I think `null` (used here) and `undefined` (which they would be as named args when not passed) are used interchangeably here?

Reply:
Good catch. Tried to remove most of these cases. But seems as I missed something in the iterations. Removed some more properties that weren't needed anymore.

Reply:
Looks good. I inline commented on a few more, that maybe could be removed. Feel free to ignore if that's not the case